tools/provision.ps1

```diff
@@ -288,6 +288,7 @@ function Install-ThirdParty {
   Try {
     foreach ($package in $packages) {
       $chocoForce = ""
+      $executionTime = 7200
```

Comment:
Should this perhaps be named `executionTimeout` instead? As it is a maximum time limit, not a fixed amount of time.